docs/reference/commandline/run.md

```diff
@@ -340,75 +340,43 @@ system's interfaces.
 $ docker run -e MYVAR1 --env MYVAR2=foo --env-file ./env.list ubuntu bash
 ```
 
-This sets simple (non-array) environmental variables in the container. For
-illustration all three
-flags are shown here. Where `-e`, `--env` take an environment variable and
-value, or if no `=` is provided, then that variable's current value, set via
-`export`, is passed through (i.e. `$MYVAR1` from the host is set to `$MYVAR1`
-in the container). When no `=` is provided and that variable is not defined
-in the client's environment then that variable will be removed from the
-container's list of environment variables. All three flags, `-e`, `--env` and
-`--env-file` can be repeated.
+The `-e`, `--env`, and `--env-file` flags allow you to set simple (non-array)
```

Comment:
I like a more active sentence like:

Use the `-e`, `--env`, and `--env-file` flags to set simple (non-array) environment variables within the container.